GUI.py

```diff
@@ -200,6 +200,8 @@ def __init__(self, master):
         self.morphemesButton = createButton(self, "Add Morphemes", 4, 3, self.inflectionalMorphemes)
         self.submitGrammarButton = createButton(self, "Submit", 4, 5, self.applyCorrection, 5)
 
+        self.labelSpeakersButton = createButton(self, "Label Speakers", 5, 0, self.labelSpeakers, lock=False) # For speaker labeling
```

Comment:
@JackPifer Make sure to add the functionality that keeps the button disabled until there is a transcription to label (look at the transcribe function in this file for info on how to do this)